scripts/diff_files

```diff
@@ -1,2 +1,4 @@
 
 core/src/Cuda/Kokkos_Cuda_Atomic_Intrinsics.hpp
+core/src/OpenMPTarget/Kokkos_OpenMPTarget_ParallelFor_Range.hpp
+core/src/OpenMPTarget/Kokkos_OpenMPTarget_ParallelFor_Team.hpp
```

Comment:
What does this do?

Reply:
I am not sure why that got added. I will remove this file.